jujugui/static/gui/src/app/app.js

```diff
@@ -1230,7 +1230,7 @@ YUI.add('juju-gui', function(Y) {
       const env = this.env;
       const db = this.db;
       const connected = this.env.get('connected');
```

Comment:
`connected` only seems to be used on line 1328 maybe just move the `this.env.get('connected')` to there? (I may have missed a usage?